src/test/java/com/wikia/webdriver/common/core/AlertHandler.java

```diff
@@ -10,7 +10,7 @@
 
 public class AlertHandler {
 
-  private AlertHandler() {
+  public AlertHandler() {
```

Comment:
please don' do this, it was private because you don't need to create an instance of this class to invoke its method, since they are static.